ant_world_test/ant_world_test.cc

```diff
@@ -63,22 +74,31 @@ int main()
     }
 
     // Enable VSync
-    glfwSwapInterval(2);
+    glfwSwapInterval(1);
+
+    glDebugMessageCallback(handleGLError, nullptr);
 
     // Set clear colour to match matlab and enable depth test
-    glClearColor(0.0f, 1.0f, 1.0f, 1.0f);
+    glClearColor(0.75f, 0.75f, 0.75f, 1.0f);
     glEnable(GL_DEPTH_TEST);
     glLineWidth(4.0);
     glPointSize(4.0);
 
-    // Create renderer
-    Renderer renderer("../libantworld/world5000_gray.bin",
-                      {0.0f, 1.0f, 0.0f},
-                      {0.898f, 0.718f, 0.353f},
-                      width, height);
+    glEnable(GL_CULL_FACE);
+    glCullFace(GL_BACK);
 
+    glEnable(GL_TEXTURE_2D);
 
-    HID::Joystick joystick;
+    // Create renderer - increasing cubemap size to improve quality in larger window
+    AntWorld::Renderer renderer(512);
+    renderer.getWorld().load("../libantworld/world5000_gray.bin",
+                             {0.0f, 1.0f, 0.0f}, {0.898f, 0.718f, 0.353f});
+
+    // Load world, keeping texture sizes below 4096 and compressing textures on upload
+    renderer.getWorld().loadObj("/home/j/jk/jk421/Documents/pier/pier_alex_smoothed_decimated_triangles.obj",
```

Comment:
I feel like this shouldn't be a hardcoded path. You could just point it to `whatever.obj` in the current folder and then people can copy or symlink it in.

Reply:
Come to think of it we could also put the file in the repo and use Git LFS, if it's not *too* huge (which I suspect it may be). You're not obliged to do a `git lfs install` on your repo so people could just download the data files if they want them.

Reply:
Yeah - I'll remove this hardcoded path. As the data in question was a sample from the drone surveying company we probably don't have the rights to publish it on our github though!